### PR TITLE
Add map TryOpenAsync to doc; Fix web auth bug

### LINKED ISSUE #516: Essentials: new Map APIs
RC3 added this PR: https://github.com/dotnet/maui/pull/5834 which makes some API additions to `Map` (Essentials).

Action: @adegeo to decide if doc updates are required.

### LINKED ISSUE #614: There's a mistake in the Android code sample
[Enter feedback here]

There's a mistake in the Web authenticator Android code sample (that's in the Get started section, after the intro "To access the WebAuthenticator functionality the following platform-specific setup is required.").

**Microsoft.Maui.WebAuthenticatorCallbackActivity should be Microsoft.Maui._Authentication_.WebAuthenticatorCallbackActivity**

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 829bc017-4a10-6ca5-0717-e1e62f59aacc
* Version Independent ID: 829bc017-4a10-6ca5-0717-e1e62f59aacc
* Content: [Web Authenticator - .NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/platform-integration/communication/authentication?tabs=android)
* Content Source: [docs/platform-integration/communication/authentication.md](https://github.com/dotnet/docs-maui/blob/main/docs/platform-integration/communication/authentication.md)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

### PR DESCRIPTION
**Pull request by adegeo**
Maps
- Add sub section detailing `TryOpenAsync`
- Add info about intent on Android

WebAuth
- Converted intent settings to code
- Fixed type namespace

Fixes #516
Fixes #614

@davidbritch

### CODE REVIEW

File: docs/platform-integration/appmodel/maps.md [hunk 1]
```diff
@@ -52,8 +89,6 @@ This section describes the platform-specific differences with the maps API.
 
 `NavigationMode` supports Bicycling, Driving, and Walking.
 
-Android uses the `geo:` Uri scheme to launch the maps application on the device. This may prompt the user to select from an existing app that supports this Uri scheme. Google Maps supports this scheme.
```
Comment (adegeo) [inline docs/platform-integration/appmodel/maps.md:55]: I moved this to the getting started section

File: docs/platform-integration/appmodel/maps.md [hunk 2]
```diff
@@ -13,9 +13,40 @@ This article describes how you can use the .NET Multi-platform App UI (.NET MAUI
...
+## Get started
+
+To access the browser functionality, the following platform-specific setup is required.
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+Android uses the `geo:` Uri scheme to launch the maps application on the device. This may prompt the user to select from an existing app that supports this Uri scheme. Google Maps supports this scheme.
+
+In the _Platforms/Android/AndroidManifest.xml_ file, add the following `queries/intent` nodes the `manifest` node:
```
Comment (davidbritch) [inline docs/platform-integration/appmodel/maps.md]: "add the following `queries/intent` nodes to the `manifest` node"

**Review by davidbritch — APPROVED** [review]
Approved but one tiny typo to fix.

File: docs/platform-integration/appmodel/maps.md (hunk 2 above)
Comment (adegeo) [reply docs/platform-integration/appmodel/maps.md]:
```suggestion
In the _Platforms/Android/AndroidManifest.xml_ file, add the following `queries/intent` nodes to the `manifest` node:
```

**Pull request merged**